### PR TITLE
HW Emulation : Add an ini switch to all HOST buffer sync with device

### PR DESCRIPTION
**Pull request by hemantk-xilinx**

### CODE REVIEW

**Comment by hemantk-xilinx** [comment]
@chvamshi-xilinx Can you please review?

**Comment by chvamshi-xilinx** [comment]
ok to test

**Review by chvamshi-xilinx — APPROVED** [review]

File: src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/shim.cxx [hunk 1]
```diff
@@ -2946,8 +2946,9 @@ int HwEmShim::xclSyncBO(unsigned int boHandle, xclBOSyncDirection dir, size_t si
   }
 
   int returnVal = 0;
-  if (!xclemulation::xocl_bo_host_only(bo)) { // bypassed the xclCopyBufferDevice2Host/Host2Device RPC calls for Slave Bridge (host only buffer scenario)
+  if (!isHostOnlyBuffer(bo)) { // bypassed the xclCopyBufferDevice2Host/Host2Device RPC calls for Slave Bridge (host only buffer scenario)
     void* buffer = bo->userptr ? bo->userptr : bo->buf;
+    std::cout<<"Calling Corret Duync"<<std::endl;
```
Comment (stsoe) [inline src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/shim.cxx:2951]: @chvamshi-xilinx @hemantk-xilinx Stray print.

Comment (hemantk-xilinx) [reply src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/shim.cxx:2951]: Fixed with https://github.com/Xilinx/XRT/pull/5509

**Pull request merged**